Author: Dave Jones <dave.jones@canonical.com>
Forwarded: not-needed
Description: Tweak paths in rpi-eeprom-update to Ubuntu defaults

--- a/rpi-eeprom-update
+++ b/rpi-eeprom-update
@@ -30,8 +30,8 @@
 ENABLE_VL805_UPDATES=${ENABLE_VL805_UPDATES:-1}
 USE_FLASHROM=${USE_FLASHROM:-0}
 RECOVERY_BIN=${RECOVERY_BIN:-${FIRMWARE_ROOT}/${FIRMWARE_RELEASE_STATUS}/recovery.bin}
-BOOTFS=${BOOTFS:-/boot}
-VCMAILBOX=${VCMAILBOX:-/opt/vc/bin/vcmailbox}
+BOOTFS=${BOOTFS:-/boot/firmware}
+VCMAILBOX=${VCMAILBOX:-/usr/bin/vcmailbox}
 
 EXIT_SUCCESS=0
 EXIT_UPDATE_REQUIRED=1
